### DISCUSSION TITLE
Airpods Pro 2 (Firmware 6B34): new functionality

### DISCUSSION THREAD

**Discussion by mainrs** [d4rken-org/capod#274] [Ideas]
I just noticed that my airpods updated and I got new functionality. When my Airpods are connected to any other device and I unlock or interact with an Apple device, I get a notification that my Airpods are connected to another non-apple device and if I'd like to connect them to my Apple device.

It's basically a notification to make hand-over easier. Maybe the update added new stuff on the BLE stack that one could use :) Just wanted to give a heads-up if someone wants to investigate!

**Comment by HK-DDboy** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。
